x/smart-account/ante/circuit_breaker.go

```diff
@@ -49,6 +49,9 @@ func (ad CircuitBreakerDecorator) AnteHandle(
 
 // IsCircuitBreakActive checks if smart account are active and if there is a
 // selected authenticator, the function will return true is the circuit breaker is active.
+// nit: this abstraction encapsulates more than just the circuit breaker - there is
+// also the check for the selected authenticator. Should we change it to
+// ShouldExecuteAuthenticatorFlow?
 func IsCircuitBreakActive(
```

Comment:
Note: Please feel free to commit if disagreement. If PR is approved, I will change name pre-merge
```suggestion
func IsCircuitBreakActive(
```

Reply:
Here I'm a bit tentative to change it, I agree with the assessment, but am tentative to change function signatures rn. :bow:

Reply:
Np. Let's close then